docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -107,19 +102,11 @@ The statement "`No data loss`" depends on the setup not being degraded from prev
 
 == Known limitations
```

Comment:
We may want to be clear on what data loss means in this case, is it persisted data or data in the cache.